docs/src/cli/transfer-tokens.md

```diff
@@ -0,0 +1,98 @@
+# Send and Receive Tokens
+
+## Receive Tokens
+
+To receive tokens, you will need an address for others to send tokens to. In
+Solana, an address is the public key of a *keypair*. There are a variety
```

Comment:
Wondering about the italics in this paragraph. I assume italics are for first use, but "keypair", for instance, is used many times in previous pages.